src/sidebar/util/test/sentry-test.js

```diff
@@ -55,6 +63,36 @@ describe('sidebar/util/sentry', () => {
       );
     });
 
+    it('configures Sentry to only report errors that can be attributed to our code', () => {
```

Comment:
Should there be a test here for a non-matching origin?

Reply:
Sentry is responsible for applying the whitelist which our code generates. This test only checks that it is generated correctly.